.github/workflows/deploy-sdk.yml

```diff
@@ -32,12 +30,6 @@ jobs:
       - name: Install sd CLI to use later in the workflow 
         uses: kenji-miyake/setup-sd@v1
 
-      # once this issue is resolved, we can remove this workaround. https://github.com/cycjimmy/semantic-release-action/issues/159#issuecomment-1490892625
```

Comment:
issue still seems to be open, are we sure this has been fixed?

Reply:
I have not been experiencing this issue in a while which is why I removed it. 

Since adding this workaround, the github action, the tool, and all plugins have gone through multiple major version updates. Making me think that there is a chance this issue no longer exists.